image-types/rhel8/amazon-ec2.md

```diff
@@ -0,0 +1,79 @@
+# Amazon EC2 Image
+
+This image is meant to be used in [Amazon Elastic Compute Cloud (EC2)][ec2], a
+popular cloud computing platform. It has to follow Amazon’s
+[guidelines][guidelines] and [requirements][requirements] for shared AMIs.
+
+## File Format
+
+EC2 uses Amazon Machine Images (AMIs) internally, which only be created inside
+EC2. An image in a standard format (ova, vmdk, vhd/x, or raw) must be imported
+from S3 storage. **osbuild-composer** generates this image type in the VHDX
+format, because it is fairly modern, widely used, and uses less space than RAW
+images.
+
+The default image size is 6 GB, unless it has to be larger due to installed
+packages.
+
+## Architectures
+
+Only `x86_64` images are generated.
+
+## Partitioning and Booting
+
+The image specifies partitions with a MBR partition table, because that is
+required by EC2. It contains one bootable partition formatted with RHEL's
+default file system XFS.
+
+## Software
+
+`cloud-init` is included in this image, because it is required by EC2.
+
+These packages are also included, for unknown reasons:
```

Comment:
If reasons are unknown how safe it is to remove these packages?   Or are they included due to depsolving ?

Reply:
These packages are removed in the official amazon image. We don't know why, yet.

The idea behind these documents is to find all the reasoning behind these decisions.